install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -85,6 +85,24 @@ spec:
 {{- end }}
         command:
         - cilium-agent
+{{- if semverCompare ">=1.20-0" .Capabilities.KubeVersion.Version }}
+        startupProbe:
+          httpGet:
+{{- if .Values.ipv4.enabled }}
+            host: '127.0.0.1'
+{{- else }}
+            host: '::1'
+{{- end }}
+            path: /healthz
+            port: {{ .Values.healthPort }}
+            scheme: HTTP
+            httpHeaders:
+            - name: "brief"
+              value: "true"
+          failureThreshold: 12
+          periodSeconds: 10
```

Comment:
This is only executed during startup so I would suggest to run this every 2 seconds and have a failure threshold of 24.

Reply:
@aanm fixed.